products/products.html

```diff
@@ -4,7 +4,7 @@
 <head>
     <link rel="stylesheet" href="../style.css">
 
-    <link rel="product-stylesheet" href="products.css">
```

Comment:
where it says link rel it should just be stylesheet instead of product-stylesheet. :)